Ruby/lib/algorithms.rb

```diff
@@ -277,5 +277,5 @@ def iterative_factorial(number)
 
 # Write a method that takes an array and returns all its permutations.
 def permutations(array)
-  
+
```

Comment:
(not really a "typo", per se, but my editor is configured to strip whitespace, and not having extraneous whitespace is probably a good practice to follow, so i went with it)